packages/admin-stories/src/admin/mui/Link.tsx

```diff
@@ -0,0 +1,13 @@
+import { Link } from "@material-ui/core";
+import { storiesOf } from "@storybook/react";
+import * as React from "react";
+
+const Story: React.FC = () => {
+    return (
+        <Link href="https://github.com/vivid-planet/comet-admin" target="_blank">
+            Link with Default Styling
+        </Link>
+    );
+};
+
+storiesOf("@comet/admin/mui", module).add("Link", () => <Story />);
```

Comment:
@nsams 

> please always add dev stories where you tested your change

What should this story encompass? Since it's the default styling I just created a Link component

Reply:
:+1:

I suppose you tested this change during developing it. And exactly that story I want to have :)